Makefile

```diff
@@ -193,6 +193,25 @@ OTOOL    := $(GNU_HOST_TRIPLE)-otool
 EXTRA    := INSTALL="/usr/bin/install -c --strip-program=$(STRIP)"
 LIBTOOL  := $(GNU_HOST_TRIPLE)-libtool
 
+else ifeq ($(UNAME),FreeBSD)
+$(warning Building on FreeBSD)
+TARGET_SYSROOT  ?= $(HOME)/cctools/SDK/iPhoneOS13.2.sdk
+MACOSX_SYSROOT  ?= $(HOME)/cctools/SDK/MacOSX.sdk
+CC              := $(GNU_HOST_TRIPLE)-clang
+CXX            := $(GNU_HOST_TRIPLE)-clang++
+CPP           := $(GNU_HOST_TRIPLE)-clang -E
+AR              := $(GNU_HOST_TRIPLE)-ar
+LD              := $(GNU_HOST_TRIPLE)-ld 
+RANLIB    := $(GNU_HOST_TRIPLE)-ranlib   
+STRIP       := $(GNU_HOST_TRIPLE)-strip
+I_N_T       := $(GNU_HOST_TRIPLE)-install_name_tool
+NM            := $(GNU_HOST_TRIPLE)-nm
+LIPO         := $(GNU_HOST_TRIPLE)-lipo
+OTOOL    := $(GNU_HOST_TRIPLE)-otool
+EXTRA      := INSTALL="/usr/bin/install -c --strip-program=$(STRIP)"
+LIBTOOL := $(GNU_HOST_TRIPLE)-libtool
```

Comment:
Fix all this alignment

Reply:
lmao vscode breaks alignment gonna use nano